packages/cli/templates/app/app/pages/index.tsx

```diff
@@ -1,12 +1,6 @@
 import {Head, Link} from 'blitz'
 
-const modelSnippet = `model Project {
```

Comment:
We should still include the `model` step in these instructions to match the README